install/resources/mybb_theme.xml

```diff
@@ -1814,7 +1814,7 @@ div.jqi .jqisubstate .jqibuttons button{
 
 /** Impromptu End */
 ]]></stylesheet>
-		<stylesheet name="usercp.css" attachedto="usercp.php|usercp2.php|private.php" version="1808" disporder="2"><![CDATA[.usercp_nav_item {
+		<stylesheet name="usercp.css" attachedto="usercp.php|private.php" version="1808" disporder="2"><![CDATA[.usercp_nav_item {
```

Comment:
The `version` here should probably be updated, but since we plan to look at altering the theme system a bit it should be ok.

Reply:
Yeah, I cherry-picked @effone's PR and applied it; he updated template versioning in a separate PR which I did not include since we're going to remove it anyway.